doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2805,6 +2805,24 @@ Build system
 
 The issues in this section are related to :ref:`app_build_system`.
 
+.. rst-class:: v2-7-0
+
+NCSDK-28462: When using sysbuild, MCUboot signing configuration cannot be updated in an already configured project.
+
+  **Workaround:** Perform a pristine build to change any of the MCUboot signing configuration, do not update it using ``menuconfig`` and instead provide it on the command line when configuring the application if it needs setting manually.
```

Comment:
What about guiconfig and nRF Kconfig GUI? Please mention them too.

Also, according to https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/config_and_build/configuring_app/kconfig/index.html#temporary_kconfig_changes using menuconfig is equivalent to using command line, so "do not update it using ``menuconfig`` and instead provide it on the command line" in your line is contradictory.

Reply:
They are not equivalent, if you configure a project and use a command line option, it will generate the initial configuration with that set, if you use menuconfig etc. then you are changing the configuration that is already set, that is the problem because once a project is configured then this part is stuck

Reply:
By "use a command line option", you mean using the CMake option `-Dname_of_Kconfig=value`?

Reply:
yes

Reply:
Then I suggest replacing `provide it on the command line` with:
```
:ref:`provide it on the command line <cmake_options>`
```
For the sake of completeness, please mention guiconfig and nRF Kconfig GUI next to menuconfig.

Reply:
Fixed